hack/dind-cluster.sh

```diff
@@ -249,7 +249,7 @@ EOF
   # Remove formatting before use
   template="$(echo "${template}" | tr -d '\n' | sed -e 's/} \+/}/g')"
   local count
-  count="$("${oc}" --config="${kubeconfig}" get nodes \
+  count="$("${oc}" --config=${kubeconfig} get nodes \
```

Comment:
I can't get this one to fail for me, and these quotes look fine. Revert this change, please.